dist/functional-test-k8s.sh

```diff
@@ -148,6 +152,37 @@ pings() {
 	assert "docker exec --privileged flannel-e2e-test-flannel2 /bin/ping -c 5 $ping_dest1" "Host 2 cannot ping host 1"
 }
 
+check_iptables() {
```

Comment:
Should we maybe have a specific test where we explicitly garble rule order to ensure that `flanneld` corrects the ordering of the rules?

Reply:
I need to check how can I do this. Do you have an idea ?

Reply:
As I know with iptables-restore the order is garanty in the transaction and the `--wait` option prevent multiple instances of the program from running concurrently (so no concurrently changes on the iptables rules)